sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -125,6 +124,9 @@ class QueryExecution(val sparkSession: SparkSession, val logical: LogicalPlan) {
               .mkString("\t")
         }
       }
+    // SHOW TABLES in Hive only output table names, while ours outputs database, table name, isTemp.
+    case command: ExecutedCommandExec if command.cmd.isInstanceOf[ShowTablesCommand] =>
+      command.executeCollect().map(_.getString(1))
```

Comment:
because we change the output of `ShowTablesCommand`, and now the `table` column is the second column.